src/api/tooling.ts

```diff
@@ -173,6 +173,14 @@ export type CompletionsResult = {
       [name: string]: ClassDeclaration;
     };
   };
+  completions?: {
+    [component: string]: {
+      simple: boolean;
+      attribs: {
+        [attr: string]: {};
+      };
+    };
+  };
```

Comment:
see this comment:
https://github.com/jsforce/jsforce/pull/1369/files#r1365627514